django/http/request.py

```diff
@@ -371,17 +371,11 @@ def __init__(self, environ):
 
     def __getitem__(self, key):
         """
-        Django template variables cannot contain hyphens, so to allow use
-        of the headers object in templates, we also try to look up the 
-        variable with underscores. For example, use headers['foo-bar']
+        Django template variable lookup precludes hyphens. By converting
+        underscores to hyphens here, we allow the use of headers['foo-bar']
```

Comment:
Avoid use of "we". (See [Coding Style guidelines](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#coding-style) 👍)